prow/config/config.yaml

```
@@ -157,6 +165,7 @@ branch-protection:
               required_status_checks:
                 contexts:
                   [
+                    "test-centos-integration-e2e-main",
                     "test-centos-integration-release-1-3",
                     "test-ubuntu-integration-main",
```

Do we need 3 required checks here? I'm thinking that it would be enough to replace the ubuntu integration test.
```suggestion
                    "test-centos-integration-release-1-3",
                    "test-ubuntu-integration-e2e-main",
```

---

I wanted to keep Ansible main tests for sometime before removing them and we still do not have integration e2e for releases

---

Not sure I understand. This would still run the normal integration test on the release branch.
For `baremetal-operator` on line 108 we do almost the same: keep one normal integration test and replace the other with e2e integration test.

---

On BMO we do not require a test on releases so only main tests do you think here as well test on release-1.3 is not required

---

Ok, but here we actually add the e2e and keep both old integration tests. That was my point. Can we not just replace `test-ubuntu-instegration-main` with `test-ubuntu-integration-e2e-main`? We still run the integration test for the release branch

---

then I add Ubuntu e2e and remove the ansible one